src/test/java/seedu/trackbeau/model/customer/SearchContainsKeywordsPredicateTest.java

```diff
@@ -46,42 +49,44 @@ public void equals() {
     @Test
     public void test_nameContainsKeywords_returnsTrue() {
         // One keyword
-        ArrayList<List<String>> prefixArr = new ArrayList<List<String>>(Collections.nCopies(9, null));
-        prefixArr.add(0, Collections.singletonList("Alice"));
+        ArrayList<List<String>> prefixArr = new ArrayList<List<String>>(Collections
+                .nCopies(FIND_ATTRIBUTE_COUNT, null));
+        prefixArr.set(0, Collections.singletonList("Alice"));
         SearchContainsKeywordsPredicate predicate = new SearchContainsKeywordsPredicate(prefixArr);
         assertTrue(predicate.test(new CustomerBuilder().withName("Alice Bob").build()));
 
         // Multiple keywords
-        prefixArr.add(0, Arrays.asList("Alice", "Bob"));
+        prefixArr.set(0, Arrays.asList("Alice", "Bob"));
         predicate = new SearchContainsKeywordsPredicate(prefixArr);
         assertTrue(predicate.test(new CustomerBuilder().withName("Alice Bob").build()));
 
         // Only one matching keyword
-        prefixArr.add(0, Arrays.asList("Bob", "Carol"));
+        prefixArr.set(0, Arrays.asList("Bob", "Carol"));
         predicate = new SearchContainsKeywordsPredicate(prefixArr);
         assertTrue(predicate.test(new CustomerBuilder().withName("Alice Carol").build()));
 
         // Mixed-case keywords
-        prefixArr.add(0, Arrays.asList("aLIce", "bOB"));
+        prefixArr.set(0, Arrays.asList("aLIce", "bOB"));
         predicate = new SearchContainsKeywordsPredicate(prefixArr);
         assertTrue(predicate.test(new CustomerBuilder().withName("Alice Bob").build()));
     }
 
     @Test
     public void test_nameDoesNotContainKeywords_returnsFalse() {
-        ArrayList<List<String>> prefixArr = new ArrayList<List<String>>(Collections.nCopies(9, null));
+        ArrayList<List<String>> prefixArr = new ArrayList<List<String>>(Collections
+                .nCopies(FIND_ATTRIBUTE_COUNT, null));
 
         // Zero keywords
         SearchContainsKeywordsPredicate predicate = new SearchContainsKeywordsPredicate(prefixArr);
         assertFalse(predicate.test(new CustomerBuilder().withName("Alice").build()));
 
         // Non-matching keyword
-        prefixArr.add(0, Arrays.asList("Carol"));
+        prefixArr.set(0, Arrays.asList("Carol"));
         predicate = new SearchContainsKeywordsPredicate(prefixArr);
         assertFalse(predicate.test(new CustomerBuilder().withName("Alice Bob").build()));
 
         // Keywords match phone, email and trackbeau, but does not match name
```

Comment:
Is this supposed to be "Keywords match phone, email and address" instead of "Keywords match phone, email and trackbeau"?